src/test/resources/junit-platform.properties

```diff
@@ -1,3 +1,4 @@
 junit.jupiter.execution.parallel.enabled=true
-junit.jupiter.execution.parallel.config.strategy=dynamic
-junit.jupiter.execution.parallel.config.dynamic.factor=0.5
+junit.jupiter.execution.parallel.mode.default=same_thread
+junit.jupiter.execution.parallel.config.strategy=fixed
+junit.jupiter.execution.parallel.config.fixed.parallelism=4
```

Comment:
This allows me to run it on my machine, I hope it is a good compromise.

Reply:
I'd *really* like to spend more time on this before we make this change. It will slow our build down significantly and I love being able to build all of koil quickly. Let's pair on it on Monday so that we can get it sorted.